Backpack/Button/Classes/BPKButton.m

```diff
@@ -88,6 +88,11 @@ - (void)setupWithSize:(BPKButtonSize)size style:(BPKButtonStyle)style {
     self.titleLabel.textAlignment = NSTextAlignmentCenter;
     self.titleLabel.tintAdjustmentMode = UIViewTintAdjustmentModeNormal;
 
+    // Use this to get the title value if one has been set in storyboard.
+    if (self.titleLabel != nil && self.titleLabel.text != nil) {
```

Comment:
I am confused as to how this resolve the problem 🤔 Can you expand a bit?

Reply:
Yeah - it's because we set the title of the disabled ones in the interface-builder. That results in calling `setTitle: forState` or `setAttributedStyle: forState`.
But we don't support those, so our button ignores them. As a result, our button behaves as if it's icon-only and doesn't properly format the text.

Reply:
Right and the reason it thinks it's icon only is because `_title` is `nil`?

Reply:
Yeah

Reply:
@k0nserv any other suggestions of how we could fix this? Or do you think this is a reasonable approach?

Reply:
Nah I think it sounds reasonable